### PR TITLE
docs: fixing typos and grammatical errors in several documentation files

### PR DESCRIPTION
**Pull request by vyom1611**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
Fixed documentation erroes

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by josephperrott** [comment]
Adding @bob-watson to review for grammar

File: docs/BAZEL.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Bazel.
 ## Installation
 
 In order to ensure that everyone builds Angular in a _consistent_ way, Bazel
-will be installed through NPM and therefore it's not necessary to install Bazel
+will be installed through NPM, and therefore it's not necessary to install Bazel
```
Comment (bob-watson) [inline docs/BAZEL.md:13]: suggested edit: `will be installed through NPM. It's not necessary to install Bazel`

File: docs/BAZEL.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ Of course, non-hermeticity in an action can cause problems.
 At worst, you can fetch a broken artifact from the cache, making your build non-reproducible.
 For this reason, we are careful to implement our Bazel rules to depend only on their inputs.
 
-Currently we only use remote caching on CircleCI and we let Angular core developers enable remote caching to speed up their builds.
+Currently, we only use remote caching on CircleCI, and we let Angular core developers enable remote caching to speed up their builds.
```
Comment (bob-watson) [inline docs/BAZEL.md:160]: Suggested edit: Currently, we only use remote caching on CircleCI. We let Angular core developers enable remote caching to speed up their builds.

File: docs/DEVELOPER.md [hunk 3]
```diff
@@ -146,7 +146,7 @@ $ yarn lint
 ## Publishing Snapshot Builds
 
 When a build of any branch on the upstream fork angular/angular is green on CircleCI, it
-automatically publishes build artifacts to repositories in the Angular org, eg. the `@angular/core`
+automatically publishes build artifacts to repositories in the Angular org, e.g. the `@angular/core`
```
Comment (bob-watson) [inline docs/DEVELOPER.md:149]: suggestion: replace `e.g.` with `for example,`

**Review by bob-watson — CHANGES_REQUESTED** [review]
Thanks for cleaning this up.
Most of these edits look good.
I left minor comments on a couple of them that should be reviewed before this is merged.

**Comment by vyom1611** [comment]
Changes added to this branch

**Review by bob-watson — CHANGES_REQUESTED** [review]
I don't see any changes or comments made for the change requests that are marked as resolved. Please make the requested edits so I can approve this.

**Comment by vyom1611** [comment]
Hey,
Anywhere I check, this branch has the updated code, and am unable to push any commits because it already has the changes. Where can you spot the original unchanged code?

**Comment by bob-watson** [comment]
@vyom1611 , it doesn't look like you've pushed any revisions to the documentation topics up to github since Aug. 25, at least that's how I read the table at the top of https://github.com/angular/angular/pull/47254/commits.

File: docs/GITHUB_PROCESS.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ providing any of the additional information the team needs to investigate.
 # Feature request process
 
 To manage the requests we receive at scale, we introduced automation in our feature request
-management process. After we triage an issue and we identify it as a feature request, it goes
+management process. After we triage an issue, and we identify it as a feature request, it goes
```
Comment (gkalpak) [inline docs/GITHUB_PROCESS.md:20]: I don't think the comma is needed here.

File: docs/BAZEL.md [hunk 5]
```diff
@@ -87,7 +87,7 @@ For additional info and testing options, see the
 [nodejs_test documentation](https://bazelbuild.github.io/rules_nodejs/Built-ins.html#nodejs_test).
 
 - Click on "Resume script execution" to let the code run until the first `debugger` statement or a previously set breakpoint.
-- If you're debugging a test and you want to inspect the generated template instructions, find the template of your component in the call stack and click on `(source mapped from [CompName].js)` at the bottom of the code. You can also disable sourcemaps in the options or go to sources and look into ng:// namespace to see all the generated code.
+- If you're debugging a test, and you want to inspect the generated template instructions, find the template of your component in the call stack and click on `(source mapped from [CompName].js)` at the bottom of the code. You can also disable sourcemaps in the options or go to sources and look into ng:// namespace to see all the generated code.
```
Comment (gkalpak) [inline docs/BAZEL.md:90]: I don't think the comma is needed here.

**Review by gkalpak — CHANGES_REQUESTED** [review]
I confirm I also see what @bob-watson sees. The requested changes are not applied (and the PR includes unrelated changes).

@vyom1611, if you look at the [Commits][1] tab, you can see there are several commits (some of them "merge" commits or unrelated to this PR). Normally, there should be one commit that contains the relevant changes.

You can also check out [Submitting a Pull Request][2] and [Addressing review feedback][3] for more details on the suggested process. Let me know if you need any help with getting this PR into shape.

[1]: https://github.com/angular/angular/pull/47254/commits
[2]: https://github.com/angular/angular/blob/7d3df4724d2baf087453d317ab6ab893d4a7b85f/CONTRIBUTING.md#-submitting-a-pull-request-pr
[3]: https://github.com/angular/angular/blob/7d3df4724d2baf087453d317ab6ab893d4a7b85f/CONTRIBUTING.md#addressing-review-feedback

**Comment by vyom1611** [comment]
Hey @bob-watson @gkalpak , sorry for all the confusion. I have added both the review changes in new commits. Should be resolved.

**Review by gkalpak — APPROVED** [review]
Thank you for making the changes, @vyom1611 :heart:

BTW, I have fixed the commits for you (removing the "merge" commits and the unrelated commits). This should be good to go now (once @bob-watson approves) 🚀

Thank you again for working on this.
(For future PRs, you might want to go through the [submission guidelines][1] to avoid issues :smiley:)

[1]: https://github.com/angular/angular/blob/9a623c03b27e35f23d4834188880a9d669020e30/CONTRIBUTING.md#-submission-guidelines

**Review by bob-watson — APPROVED** [review]
LGTM
Thanks for the fixes!

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 0d13cc78485b0631a744024bcbb01a94b274d335.

**Pull request closed without merging**